examples/strings-io/strings_io.bal

```diff
@@ -1,32 +1,32 @@
 import ballerina/io;
 
 public function main() returns @tainted error? {
-    // Initialize the text path and the content.
+    // Initializes the text path and the content.
     string textFilePath1 = "./files/textfile1.txt";
     string textFilePath2 = "./files/textfile2.txt";
     string textFilePath3 = "./files/textfile3.txt";
     string textContent = "Ballerina is an open source programming language.";
     string[] lines = ["The Big Bang Theory", "F.R.I.E.N.D.S",
     "Game of Thrones", "LOST"];
 
-    // Write the given string to a file.
+    // Writes the given string to a file.
     check io:fileWriteString(textFilePath1, textContent);
     // If the write operation was successful, then, read the content as a string.
```

Comment:
```suggestion
    // If the write operation was successful, then, reads the content as a string.
```

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/e7dc5ab61d2e03737d6158d4fd59a02371e55d84